environment-dev.yml

```diff
@@ -4,6 +4,7 @@ channels:
 dependencies:
   - python =3.9
   - pip
+  - git-lfs
```

Comment:
The [conda-forge distribution](https://anaconda.org/conda-forge/git-lfs) has the latest versions, [PyPI releases](https://pypi.org/project/git-lfs/) are outdated/unmaintained for this project.